packages/odyssey-react/src/components/Icon/ArrowDown.tsx

```diff
@@ -22,20 +22,18 @@ export interface Props
   title?: string;
 }
 
-const Caret = forwardRef<SVGSVGElement, Props>((props, ref) => {
+const ArrowDown = forwardRef<SVGSVGElement, Props>((props, ref) => {
   const omitProps = useOmit(props);
   return (
     <Icon ref={ref} {...omitProps}>
       <svg viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
         <path
-          fillRule="evenodd"
-          clipRule="evenodd"
-          d="M7.63967 8.06464C7.29376 8.43526 6.70625 8.43526 6.36032 8.06464L2.73967 4.18536C2.39376 3.81474 1.80625 3.81474 1.46032 4.18536L1.43223 4.21547C1.11846 4.55165 1.11846 5.07335 1.43223 5.40953L6.35589 10.6849C6.70344 11.0573 7.29436 11.0552 7.63934 10.6804L12.5605 5.33417C12.8759 4.99164 12.8644 4.46128 12.5347 4.13263C12.1876 3.78665 11.623 3.79684 11.2886 4.15514L7.63967 8.06464Z"
+          d="M10.5908 4.91005L7.19206 8.98851C7.09211 9.10845 6.90789 9.10845 6.80794 8.98851L3.40922 4.91004C3.27353 4.74721 3.38932 4.5 3.60128 4.5L10.3987 4.5C10.6107 4.5 10.7265 4.74721 10.5908 4.91005Z"
           fill="currentColor"
         />
       </svg>
     </Icon>
   );
 });
 
-export default Caret;
+export default ArrowDown;
```

Comment:
would be great to add an explicit `IconArrowDown.displayName = "IconArrowDown"` statement in our generate as well